mcs/tools/mono-api-diff/mono-api-diff.cs

```diff
@@ -451,7 +498,9 @@ public override void LoadData (XmlNode node)
 			}
 
 			if (atts == null || atts.Name != "namespaces") {
-				Console.Error.WriteLine ("Warning: no namespaces found!");
+#if !EXCLUDE_DRIVER
+				Console.Error.WriteLine (@"Warning: no namespaces found for {name}");
+#endif
```

Comment:
Leaving this here as there are only 2 instances of writing to the console - and these aren't very important messages at all.

Reply:
In this case, I didn't want to do some hectic abstraction or some advance mechanism just for two boring messages that are probably going to be ignored anyway since they are potentially valid cases. Sure, and assembly without namespaces is weird, but my just be an empty assembly. And sure, an empty namespace is pointless, but is it really bad?